engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1189,7 +1169,6 @@ public void testSimpleTrippleQuote() throws Exception {
   }
 
   @Test
-  @Ignore // enable CodeLocationsTest: "be correct in the presence of comments"
```

Comment:
This was an edge case bug; I added a test case for this in `tests/parse.rs`, and fixed it in the lexer. I have a [refactor planned](https://www.pivotaltracker.com/story/show/183797744) to reduce the risk of bugs related to line-segmentation, which is currently more complicated than necessary.